content/christine_belzie/create-a-url-using-slugs.md

```diff
@@ -0,0 +1,71 @@
+  - "introduction-to-the-web"
+  - "paths/web-development"
+---
+
+## Introduction 
+
+Picture this, you're scrolling through the internet, find a cookie recipe, and click on the Share button to send this link to your friend:
+(https://downshiftology.com/recipes/gluten-free-chocolate-chip-cookies/) 
+
+You see `gluten-free-chocolate-chip-cookies` after the backslash (“/”)? This is called a **slug**. It is a part of the URL that identifies a specific page in a website to make it easier for people to read. In other words, slugs explain the content on the page. In this article, you will learn the importance of slugs in your SEO strategy, how to create them, and best practices.   
+
+
+## Why Are Slugs Important to SEO Strategy?
+
+Even though slugs won't have a direct impact on how your content ranks on the web, they do influence a visitor's decision on whether to view your content or not. Here's how:
+
+1. **Increases Search Ranking:** Cannibalization is a common issue for content creators. It's when most of the blog posts, articles, and other pieces of content on a website share the same keyword. This makes it difficult for people to find them on your site. Slugs solve this issue by putting a unique name like a title or heading, thus, making it easier for people to find a person's work. Think of this as the names of the folders on your Google Drive or OneDrive account. 
+
+2.  **Improves Shareability:** 
+Adding slugs to your content's urls makes them shorter, thus, making it easier for your viewers to copy, paste, and send them to other people. 
+Now, let's create a slug! 
+
+## How to Create a Slug 
+
+Consider using the following methods to create a slug:
+
+### Method 1: Manually 
+
+1. **Think of a short yet descriptive name:**  We recommend making it easy for people to remember. For example, let's say a person wanted to create a slug for the blog post titled "How to Make Your Issue Requests Come True".
+
+2. **Place hyphens ("-") in between the parts of your name:**  Spaces can't be used in a URL, so you'd have to replace them with symbols like "%20", which makes things really hard to read. By using hyphens, your URL would be easier for people to read. Here's how it would look with the blog post referenced in the previous step: `how-to-make-your-issues-requests-come-true`.
+
+3. **Done:** Your URL is now SEO-Friendly 
+
+### Method 2: Use an URL Slug Generator 
+
+Creating a URL slug for your content can sometimes very be time-consuming, so we suggest using a tool like [slugify.online](https://slugify.online/)  to help you finish quickly:
+
+1. Type your content's title in the textbox titled **INPUT STRING**. Make sure that the **Separate with a dash (-)** option is highlighted. 
+
+![Step 1 of URL slug](https://github.com/Codecademy/ugc/main/content/christine_belzie/step1-of-slug.png)
+
+2. Click on **Slugify(Generate Slug)**
+
+![Step 2 of URL slug](https://github.com/Codecademy/ugc/main/content/christine_belzie/step2-of-slug.png)
```

Comment:
@CBID2 Would you be able to edit the arrows and markings in the image? If you're on a Mac, there should be a toolbar that you can access to help draw arrows and shapes with no fill

Reply:
Sure